lua/wire/client/text_editor/wire_expression2_editor.lua

```diff
@@ -1,5 +1,8 @@
 local Editor = {}
 
+-- Global Editor Type Checker
+GetEditorType = nil
```

Comment:
This is real nasty. Needing to get the editor type elsewhere should be handled in a different way (eg. passing the editor panel as an argument when creating the color picker panel).